components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
+  }
+
+  // Handle "Enter" key to open/close collapsible
+  function handleKeyDown(e) {
+    if (e.keyCode === 13) handleActive();
+  }
+
   return (
-    <CollapsibleWrapper>
+    <CollapsibleWrapper
+      tabIndex={0}
```

Comment:
Adding focus to the container rather than using the `tabIndex` prop for ReactCollapsible for styling issues.